Patches/Blastech Industries/Patch_SWFactions.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+<Operation Class="PatchOperationFindMod">
+    <mods>
+        <li>Star Wars - Factions (Continued)</li>
+        <li>Star Wars - Factions</li>
+    </mods>
+	
+	
+	<match Class="PatchOperationSequence">
+	<operations>
+
```

Comment:
None of the weapons currently use ammo, so these operations don't seem to be working. It's possible that they're happening before the main patch and are thus being overwritten. I'd put them into the same file as the guns at the bottom.

Reply:
The ammo is going to be from the Star Wars Factions CE patch. It is currently its own mod but I will try to add it to CE.

Reply:
As per my comment on #354, the Star Wars factions ammo should be implemented as PatchOperations within the Star Wars Factions patches folder. 

Then, in the `PatchOperationFindMod` part of the ammo patch file, add a reference to `[JDS] Star Wars BlasTech Industries`, so that the ammo patches are dynamically loaded if either JDS or the Star Wars Faction mods are active.

Reply:
That makes sense. I moved the ammo patch to the Star Wars - Factions folder